docusaurus/docs/Android/03-ui/03-chatui.md

```diff
@@ -1,54 +1,118 @@
+In order to define custom set of reactions for your chat app, you need to pass the the following data to the `ChatUI.supportedReactions` property:
+```kotlin 
+val loveDrawable = ContextCompat.getDrawable(this, R.drawable.ic_reaction_love)!!
+val loveDrawableSelected = ContextCompat.getDrawable(this, R.drawable.ic_reaction_love)!!.apply { setTint(Color.RED) }
+val supportedReactionsData = mapOf(
+    "love" to SupportedReactions.ReactionDrawable(loveDrawable, loveDrawableSelected)
+)
+ChatUI.supportedReactions = SupportedReactions(this, supportedReactionsData)
+```
+As a result, there will only be a _love_ reaction available in the chat, and when selected the reaction icon will have a red tint.
 
-## Navigator
+| Normal state - available reactions | Active state - reaction selected |
+| --- | --- |
+|![Light_mode](../assets/chat_ui_custom_reaction.png)|![Dark_mode](../assets/chat_ui_custom_reaction_active.png)|
 
-To display some screens available in the SDK, it is necessary to navigate to them.
-The SDK does that using the ChatNavigator and it navigates to:
-- `AttachmentGaleryActivity`: To display the gallery of pictures
-- After a click in a link, to the destination of the link (the user leaves the chat).
+### Custom MIME type icons
 
-It is possible to add a custom navigator to `ChatUI` by adding a new `ChatNavigator`.
+When possible SDK displays thumbnails for image files. Mime type icons are displayed for other files at `MessageListView` as attachments icons and at files gallery screen. 
+By default SDK provides built-in MIME type icons for the most popular file types, and displays a generic file icon for others.
 
-A user can instantiate a `ChatNavigator` and provide its custom implementation of `ChatNavigationHandler`:
+In order to customize MIME icons, you need to override `ChatUI.mimeTypeIconProvider` in the following way:
 
 ```kotlin
-public interface ChatNavigationHandler {
-    /**
-     * Attempt to navigate to the given [destination].
-     *
-     * @return true if navigation was successfully handled.
-     */
-    public fun navigate(destination: ChatDestination): Boolean
+ChatUI.mimeTypeIconProvider = MimeTypeIconProvider { mimeType ->
+    if (mimeType == null) {
+        R.drawable.stream_ui_ic_file
+    }
+
+    when {
+        // special icon for XLS files
+        mimeType == "application/vnd.ms-excel" -> R.drawable.ic_file_xls
+        // generic icon for audio files
+        mimeType.contains("audio") -> R.drawable.ic_file_mp3
+        // generic icon for video files
+        mimeType.contains("video") -> R.drawable.ic_file_mov
+        // generic icon for other files
+        else -> R.drawable.stream_ui_ic_file
+    }
 }
 ```
 
-Following there's a simple example of a customization of destination:
+### Transforming avatar bitmap with AvatarBitmapFactory
+
+It is possible to override `AvatarBitmapFactory` so the avatars will
+be generated accordingly to the new configuration. It is possible to configure
```

Comment:
`You can configure the user`... You we don't write `It is possible` twice

Reply:
Fixed